server/graphql/v2/object/Update.js

```diff
@@ -19,7 +21,7 @@ const Update = new GraphQLObjectType({
       slug: { type: new GraphQLNonNull(GraphQLString) },
       userCanSeeUpdate: {
         description: 'Indicates whether or not the user is allowed to see the content of this update',
-        type: new GraphQLNonNull(GraphQLBoolean),
+        type: GraphQLBoolean,
```

Comment:
Having that non-null was actually welcome, there's no value in having that nullable. If it's crashing you can do:

```
return Boolean(req.remoteUser && req.remoteUser.canSeeUpdates(update.CollectiveId));
```